library/ssl_misc.h

```diff
@@ -1415,6 +1422,13 @@ int mbedtls_ssl_tls13_start_handshake_msg( mbedtls_ssl_context *ssl,
 int mbedtls_ssl_tls13_finish_handshake_msg( mbedtls_ssl_context *ssl,
                                             size_t buf_len,
                                             size_t msg_len );
+/*
+ * Update checksum with handshake message
+ */
```

Comment:
it has been added by #4919

Reply:
mbedtls_ssl_tls13_fetch_handshake_msg is rename it to mbedtls_ssl_tls1_3_fetch_handshake_msg and add it into #4977